### PR TITLE
fix: multi editor bottom cut off

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #39348

Before:
Screenshot_1
After:
Screenshot_2

### CODE REVIEW

**Review by gikf — APPROVED** [review]
LGTM :+1: 

Checked on Gitpod with and without `SHOW_UPCOMING_CHANGES` set to `true` in `.env` file. Issue no longer appears.

**Review by Nirajn2311 — APPROVED** [review]
LGTM 👍 . Nothing was broken from in Mac.

**Comment by Sembauke** [comment]
> LGTM 👍
> 
> Checked on Gitpod with and without `SHOW_UPCOMING_CHANGES` set to `true` in `.env` file. Issue no longer appears.

This issue only appears in the multiple file editor though.

**Comment by nhcarrigan** [comment]
> This issue only appears in the multiple file editor though.

You are correct, but when changes could potentially impact the current editor it never hurts to test. Just to make sure we don't break production. 🙂

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

I double-checked and, as expected, the editor that's used in production is unchanged.  Nice work!

File: client/src/templates/Challenges/classic/DesktopLayout.js [hunk 1]
```diff
@@ -84,17 +84,13 @@ class DesktopLayout extends Component {
...
                 <ReflexElement flex={1} {...reflexProps} {...resizeProps}>
-                  {
-                    <Fragment>
-                      {showUpcomingChanges && !hasEditableBoundries && (
-                        <EditorTabs />
-                      )}
-                      {editor}
-                    </Fragment>
-                  }
+                  {<Fragment>{editor}</Fragment>}
```
Comment (Nirajn2311) [inline client/src/templates/Challenges/classic/DesktopLayout.js:93]: I just realised this 😅 , those curly braces around `Fragment` is not required.

Comment (ojeytonwilliams) [reply client/src/templates/Challenges/classic/DesktopLayout.js:93]: Oops, I didn't spot that either. They don't actually do anything in this case, right?

Comment (Sembauke) [reply client/src/templates/Challenges/classic/DesktopLayout.js:93]: I'll change that in my next pr

**Pull request merged**